docs/build/blast-tutorial-1.md

```diff
@@ -3,29 +3,18 @@ title: Create Counter project
 id: create-counter
 ---
 
-This short tutorial walks you through setting up your project and creating a simple counter that increments upon request using **Cudos Blast**.
+This short tutorial creates a simple counter that increments upon request using **Cudos Blast**.
 
-It assumes that you have installed **Cudos Blast** globally. 
+1. Check your in the 'counter project' you created in [Cudos Blast](/docs/build/blast-install-1.md).
 
-1. Create a new folder for the project and initialise Blast.
-
-```bash
-mkdir myCudosDapp
-cd myCudosDapp
-blast init
-Success! Sample project initialized in /Users/username/projects/myCudosDapp
+```shell
+$ pwd
 ```
+2. Open the project in your favourite IDE. (We're using Visual Studio Code on a Macbook for this tutorial)
```

Comment:
MacBook is a bit irrelevant. I would would phrase this as "if you're not sure what to use, we recommend Visual Studio Code". The instructions don't depend on which IDE you're using.

Reply:
Done